### ISSUE TITLE
Navbar not right aligned

### ISSUE THREAD

**Issue by marcgauch** [bootstrap-vue/bootstrap-vue#6829]
### Documentation issue

- [ ] Reporting a typo
- [x] Reporting a documentation bug
- [ ] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?
https://bootstrap-vue.org/docs/components/navbar

### Additional context or description
Using Boostrap-vue 2.21.2 using the default Navbar copied from the example the navbar is not right aligned. (Using Firefox 94.0, Opera 80.0.4170.63 and Edge on Windows 10)

![grafik](https://user-images.githubusercontent.com/34353267/140044472-36f4aaa0-9945-445a-9a0c-1e4b201e4453.png)

Changing 
```javascript
<!-- Right aligned nav items -->
<b-navbar-nav class="ml-auto">
```
to
```javascript
<!-- Right aligned nav items -->
<b-navbar-nav class="ms-auto">
```
Seems to fix this.

![grafik](https://user-images.githubusercontent.com/34353267/140044817-1438ba40-4b7b-4cbd-9048-d24506767895.png)

(Source: https://stackoverflow.com/a/65562241)
<details>
<Summary>package.json</Summary>

  ```javascript
{
  "name": "frontend",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint"
  },
  "dependencies": {
    "bootstrap": "^5.1.3",
    "bootstrap-vue": "^2.21.2",
    "core-js": "^3.6.5",
    "vue": "^2.6.14",
    "vue-router": "^3.2.0",
    "vuex": "^3.4.0"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "~4.5.0",
    "@vue/cli-plugin-eslint": "~4.5.0",
    "@vue/cli-plugin-router": "~4.5.0",
    "@vue/cli-plugin-vuex": "~4.5.0",
    "@vue/cli-service": "~4.5.0",
    "@vue/eslint-config-prettier": "^6.0.0",
    "babel-eslint": "^10.1.0",
    "eslint": "^6.7.2",
    "eslint-plugin-prettier": "^3.3.1",
    "eslint-plugin-vue": "^6.2.2",
    "prettier": "^2.2.1",
    "vue-template-compiler": "^2.6.11"
  }
}
```
</details>

**Comment by Hiws** [comment]
BootstrapVue doesn't support Bootstrap 5.

It sounds like you've installed the incorrect version.

**Comment by marcgauch** [comment]
True. It does install Bootstrap 5. But i've installed it with ```npm install vue bootstrap bootstrap-vue```. I deleted *node_modules*, *package-lock.json* and the two bootstrap entries in *package.json*. Ran ```npm install``` and ```npm install vue bootstrap bootstrap-vue```. Still the same.

This isn't a documentation issue - but what am I doing wrong?

**Comment by Hiws** [comment]
If you don't specify a version when doing `npm install bootstrap` it will just install the latest version.
You need to specify it. `npm install bootstrap@4.5.3`

**Comment by marcgauch** [comment]
This isn't described here:
https://bootstrap-vue.org/docs#using-module-bundlers

Should I open a new Issue for that?

**Comment by Hiws** [comment]
There's like 3 PRs open to fix it. 
But the library isn't being maintained currently, so no merges and updates are being done.
So i wouldn't bother.

**Comment by janswist** [comment]
Also had that issue. Spent too much time thinking where I made the error. Thanks for pointing it out!
